### ISSUE TITLE
Defining types for array keys

### ISSUE THREAD

**Issue by berkayunal** [klesun/deep-assoc-completion#91]
Hi

Is it possible to define the type for an array key with the plugin?

`/** @var  $data['brand_detail'] = \App\Shop\Brand\Models\BrandModel() */`

So when you can autocomplete it like below

`$data['brand_detail']->title`

I can do this by having it like below. But it needs additional step.

```
/** @var \App\Shop\Brand\Models\BrandModel $__detail */
$__detail = $data['brand_detail'];
```

Thanks for the plugin! Love it!

**Comment by klesun** [comment]
Hi. Sounds like a useful feature.
Currently you can only add type for the single key this way:
![image](https://user-images.githubusercontent.com/5202330/58624442-c4956400-82bf-11e9-946f-8d2939fd4c12.png)
But I do get that this is pretty confusing way to type it. I'll add the ability to write `@var $data['brand_detail'] = ...` in the next release.

**Comment by klesun** [comment]
soon
![image](https://user-images.githubusercontent.com/5202330/58668265-f17d6180-8327-11e9-8d3d-57ece26793ba.png)

**Comment by klesun** [comment]
Released in [2019.05.31.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/63310)
